### PR TITLE
BLD: remove epicscorelibs dependency

### PR DESCRIPTION
**Pull request by tangkong**
As noted by @ZLLentz in https://github.com/pcdshub/pcds-envs/pull/302#issuecomment-1758118321

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,6 @@ requirements:
   - python >=3.9
   - pyqt >=5
   - pyca =3.2.0
```
Comment (ZLLentz) [inline conda-recipe/meta.yaml:25]: This line is also pretty suspicious, it should maybe be >= instead of fixed
```suggestion
  - pyca >=3.2.0
```

**Review by ZLLentz — DISMISSED** [review]
lgtm

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
I'm switching pmgr in our spreadsheet to be the kind of repo that doesn't expect a docs build and manually moving the "required" element from the docs check here

**Comment by ZLLentz** [comment]
Ok, now we will be able to pull in pmgr PRs without admit override on the docs build failing (no docs folder)

**Comment by tangkong** [comment]
I had just done the manual branch protection fix.  Great minds think alike 👍

**Pull request merged**